dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/bundle/TrackerBundle.java

```diff
@@ -262,4 +262,9 @@ public ProgramStageInstance getProgramStageInstance( String event )
     {
         return getPreheat().getEvent( event );
     }
+
+    public org.hisp.dhis.relationship.Relationship getRelationship( String relationship )
```

Comment:
Can you please move this into the preheat. The implementation shows that it actually belongs in there as its just delegating to it. If you have the bundle, you also have the preheat. So this should easy for client code.

Reply:
I put it here because it make sense to treat Relationship as any other entity.
We either remove all the others delegation for TEI, Enrollment or Event or we maintain all of them.